redshells/model/graph_convolutional_matrix_completion.py

```diff
@@ -314,6 +319,26 @@ def _predict(cls, user_ids: List, item_ids: List, with_user_embedding, graph: Gr
         predictions = np.clip(predictions, dataset.rating()[0], dataset.rating()[-1])
         return predictions
 
+    @classmethod
+    def _get_user_feature(cls, user_ids: List, item_ids: List, with_user_embedding,
+                          graph: GraphConvolutionalMatrixCompletionGraph, dataset: GcmcGraphDataset,
+                          session: tf.Session) -> np.ndarray:
+        if graph is None:
+            RuntimeError('Please call fit first.')
+
+        rating_adjacency_matrix = dataset.train_rating_adjacency_matrix()
+        user_indices, item_indices = dataset.to_indices(user_ids, item_ids)
+        if not with_user_embedding:
+            user_indices = np.array([0] * len(user_indices))  # TODO use default user index.
```

Comment:
`TODO` is remaining. Do you fix this in this PR？ or you can post it to issue and fix it later

Reply:
I've just created the issue ticket about this TODO.